Standards/scs-XXXX-v1-volume-type-standard.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Volume Type Standard
+type: Standard
+status: Draft
+track:  IaaS
+---
+
+## Introduction
+
+A volume is a virtual drive that is to be used by an instance (i. e., a virtual machine). With OpenStack,
+each volume is created per a type that determines basic features of the volume as provided by the backend,
+such as encryption, replication, or quality of service. As of the writing of this document, presence or absence of these
+features can not be discovered with full certainty by non-privileged users via the OpenStack API.
+
+### Glossary
+
+The following special terms are used throughout this standard document:
+| Term | Meaning |
+|---|---|
+| volume | OpenStack ressource, virtual drive which usually resides in a network storage backend |
+| volume feature | A certain feature a volume can possess |
+| aspect | part of a volume type that will activate a corresponding feature in a created volume |
+| AZ | Availability Zone |
+| Volume QoS | Quality of Service object for Volumes |
+
+## Motivation
+
+As an SCS user, I want to be able to create volumes with certain common features, such as encryption or
+replication, and to do so in a standardized manner as well as programmatically.
+This standard outlines a way of formally advertising these common aspects for a volume type to
+non-privileged users, so that the most suitable volume type can be discovered and selected easily — both by
+the human user and by a program.
+
+## Design Considerations
+
+All considerations can be looked up in detail in the [Decision Record for the Volume Type Standard.](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0111-v1-volume-type-decisions.md)
+
+### Systematic Description of Volume Types
+
+To test whether a deployment has volume types with certain aspects, the discoverability of the parameters in the volume type has to be given. As for the time this standard is created, there is no way for users to discover all aspects through OpenStack commands. Therefore the aspects, that are fulfilled within a volume type, should be stated in the beginning of the **description** of a volume type in the following manner:
+
+`[scs:aspect1, aspect2, ..., aspectN]...`
+
+The mentioned aspects MUST be sorted alphebetically and every aspect should only be mentioned to the maximal amount of one.
+
+### Standardized Aspects
+
+The following table shows, which aspects are considered in this standard. The last column shows how the description of the volume type has to be adjusted, if the aspect is fulfilled:
+
+| Aspect | Requirement | standardized description | comment |
+| ---- | ---- | ------ | ------ |
+| Encryption | **Recommended** | **"[scs:encrypted]"** | used to encrypt the volume |
```

Comment:
```suggestion
| Encryption | **Recommended** | **"[scs:encrypted]"** | volume is encrypted |
```
to make it consistent with the other comment, and to avoid confusion about the meaning of "used to"  (what is used? by whom?)